translations-script/export.js

```diff
@@ -4,39 +4,70 @@ const fs = require('fs');
 const _ = require('lodash');
 const Excel = require('exceljs');
 
-function process(ws, path, enObj, gaObj) {
-  if (!_.isObject(enObj)) {
-    const enVal = enObj.replace(/\n/g, '\\n');
-    const gaVal = _.get(gaObj, path, '');
+function process(ws, path, en, ht, ru, bn, ko, zh) {
+  if (!_.isObject(en)) {
+    const enVal = en.replace(/\n/g, '\\n');
+    const htVal = _.get(ht, path, '');
+    const ruVal = _.get(ru, path, '');
+    const bnVal = _.get(bn, path, '');
+    const koVal = _.get(ko, path, '');
+    const zhVal = _.get(zh, path, '');
+
     ws.addRow({
       context: path,
       english: enVal,
-      gaeilge: (gaVal && gaVal.replace(/\n/g, '\\n')) || ''
+      haitian: htVal,
+      russian: ruVal,
+      bengali: bnVal,
+      korean: koVal,
+      chinese: zhVal
```

Comment:
Won't the other languages only be processed when `en` is not an object? Is that expected?